Editor/Editor.vb

```diff
@@ -421,6 +426,7 @@ Public Class Editor
     Private Function Save(filename As String) As Boolean
         Try
             m_fileWatcher.EnableRaisingEvents = False
+            m_fileWatcherQuest.EnableRaisingEvents = False

```

Comment:
Regardless... if `m_fileWatcherQuest` is only monitoring the Quest app directory, there's no need to deactivate it here when a game file is saved